client/components/domains/register-domain-step/style.scss

```
@@ -378,3 +378,11 @@ body.is-section-signup.is-white-signup {
 		}
 	}
 }
+
+.search-component .search-filters__dropdown-filters button {
```

```suggestion
.start-writing .search-component .search-filters__dropdown-filters button {
```
Should we be more specific with this rule, just in case?

---

I intentionally made it broad, [from the domain upsell designs](https://href.li/?pebzTe-Fx-p2#comment-1131), we should also have that border hidden